src/abstractdataframe/io.jl

```diff
@@ -114,12 +115,15 @@ function _show(io::IO, ::MIME"text/html", df::AbstractDataFrame;
     write(io, "</thead>")
     write(io, "<tbody>")
     if summary
-        omitmsg = if mxcol < size(df, 2)
-                      " (omitted printing of $(size(df, 2)-mxcol) columns)"
-                  else
-                      ""
-                  end
-        write(io, "<p>$(digitsep(size(df, 1))) rows × $(digitsep(ncol(df))) columns$omitmsg</p>")
+        header = "$(digitsep(size(df, 1))) rows × $(digitsep(ncol(df))) columns"
+        if mxcol < size(df, 2)
+            header *= " (omitted printing of $(size(df, 2)-mxcol) columns)"
+        end
+        write(io, "<p>$header</p>")
+        if mxcol == 0
+            write(io, "</tbody></table>")
+            mxcol == 0 && return
```

Comment:
we did this test above so this should be enough I think:
```suggestion
            return
```

Reply:
Good catch - a sloppy refactoring of some earlier code. I'll remove it.